nilearn/plotting/find_cuts.py

```diff
@@ -196,6 +196,10 @@ def find_cut_slices(img, direction='z', n_cuts=12, spacing='auto'):
     orig_data = np.abs(img.get_data())
     this_shape = orig_data.shape[axis]
 
+    if not isinstance(n_cuts, numbers.Number):
+        raise ValueError("The number of cuts should be a strictly positive integer. "
+                         "You provided n_cuts=%s " % n_cuts)
```

Comment:
The error message does not match the condition: Here you are interested in the type. "should" is not strong enough, this is a hard constraints. Maybe something like:
`The number of cuts (n_cuts) must be an integer greater or equal than 1. You provided a value of type %s.`